static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -0,0 +1,261 @@
+@import '../_mixins';
+
+$labour-liverpool-header-element-opacity: .85;
+
+.immersive-header-container--labour-liverpool {
+    min-height: gs-height(6);
+
+    .immersive-main-media__media {
+        // fallback for IE not supporting object-fit: cover
+        @include mq($from: desktop) {
+            max-height: gs-height(8);
+        }
+    }
+
+    .content__wrapper--standfirst {
+        background-color: #ffffff;
+        padding: 0;
+
+        // necessary for Safari
+        &.l-side-margins {
+            height: auto;
+        }
+    }
+}
+
+.content__standfirst--labour-liverpool-article {
+    @include fs-headline(3);
+    color: darken($news-support-4, 25%);
+    margin-top: $gs-baseline;
+
+    .membership-cutout {
+        float: left;
+        width: 140px;
+        margin: 0 $gs-gutter/-2;
+
+        @include mq($from: mobile, $until: tablet) {
+            display: none;
+        }
+    }
+
+    > p:first-of-type {
+        display: inline;
+    }
+}
+
+.labour-liverpool-quotes__svg {
+    fill: $news-support-4;
+    width: 30px;
+    height: 18px;
+    padding-right: $gs-gutter/4;
+}
+
+.labour-liverpool-headline {
+    position: absolute;
+    top: gs-height(6);
+    max-width: gs-span(8);
+
+    @include mq($from: wide) {
+        left: 5%;
+    }
+}
+
+.labour-liverpool-headline__content {
+    @include fs-headline(5);
+    position: absolute;
+    font-weight: 200;
+    color: #ffffff;
+    margin: $gs-baseline 0;
+    padding-left: $gs-gutter;
+    width: gs-span(4);
+
+    @include mq($from: tablet) {
+        @include fs-headline(8);
+        font-weight: 200;
+        width: gs-span(7);
+        margin: $gs-baseline $gs-gutter*2;
+    }
+}
+
+.labour-liverpool-headline__background {
+    position: absolute;
+    left: -$gs-gutter*1.5;
+    height: 130px;
+    transform: rotate(180deg);
+
+    @include mq($from: tablet) {
+        left: auto;
+    }
+}
+
+.inline-labour-liverpool-header-bg__svg {
+    fill: $news-main-2;
+    opacity: $labour-liverpool-header-element-opacity;
+}
+
+.series-identity {
+    position: absolute;
+    max-width: gs-span(3.5);
+    top: $gs-baseline*3;
+
+    @include mq($from: tablet) {
+        top: 0;
+    }
+}
+
+.series-identity__background {
+    position: absolute;
+    top: $gs-baseline/-2;
+    width: gs-span(3.5);
+}
+
+.inline-labour-liverpool-identity-bg__svg {
+    fill: $news-main-1;
+    opacity: $labour-liverpool-header-element-opacity;
+    width: 100%;
+    height: 100%;
+}
+
+.series-identity__links {
+    position: absolute;
+    left: $gs-gutter;
+    top: $gs-baseline/2;
+    width: gs-span(3.5);
+
+    &:hover {
+        text-decoration: none;
+    }
+}
+
+.series-identity__title, .series-identity__byline {
+    @include fs-header(2);
+    line-height: 19px;
+    color: #ffffff;
+}
+
+.content--labour-liverpool-article__headline-background {
+    position: absolute;
+    //TODO: Need a designer to fix the SVG so that we don't need to position it in the CSS
```

Comment:
Not a blocker, but make sure you talk to @blongden73 or another designer to fix these svgs!